src/components/legal/TermsOfService.tsx

```diff
@@ -50,24 +52,42 @@ export const TermsOfService: React.FC = () => {
 
                 <div className="mt-10 ">
                     <h1 className="text-4xl text-blue-light-contrast font-bold">Minimum Age Requirement</h1>
-                    <p className="text-xl mt-4 leading-9 font-light">To protect the privacy of younger people it is very important that we implement an age restriction of <b>13 and
-                    over</b> to join Flybywire Simulation services such as this website and Discord or any other communication
-                    platform, other platforms may be subject to their own conditions. By using Flybywire Simulations services, you
-                    agree that you are 13 years of age and the minimum age of digital consent in your country.
-                    Flybywire Simulations reserve the right to remove users under the age of 13 and minimum age of digital consent
-                    in any given case from communication platforms.</p>
+                    <p className="text-xl mt-4 leading-9 font-light">
+                        To protect the privacy of younger people it is very important that we implement an age restriction of
+                        <b>
+                            13 and
+                            over
+                        </b>
+                        {' '}
+                        to join Flybywire Simulation services such as this website and Discord or any other communication
```

Comment:
Some funny business going down here. Some text is being split with { ' ' } as a replacement for space.

Reply:
This is normal so that the tags can be on their own lines, otherwise there is no spacing